guardian/cdk-cli
# refactor: Store both kebab and pascal versions of name

## What does this change?

This PR updates the config items of the `new` and `migrate` commands so that the `appName` and `stackName` values are now objects containing both the kebab-case and PascalCase versions of the name. This makes the case being used around the tool more explicit and saves repeating the conversion.

In line with this change, a number of other changes have been made:

* Define a `Name` interface
* Add a `pascalCase` function
* Convert the app and stack names provided to PascalCase rather than assuming they will already be in that form
* Replace usages of the names with the correct version as required

## How to test

Run the `new` and `migrate` commands, providing the app and stack args with different cases, and see that the files are correctly rendered in all cases

## How can we measure success?

The code is more readable and repeated actions are removed
